docker-compose-reorg.yml

```diff
@@ -0,0 +1,284 @@
+          --allow-insecure-unlock \
+          --rpc \
+          --rpcaddr "0.0.0.0" \
+          --rpcapi "personal,eth,web3,net,admin,debug,db" \
+          --rpccorsdomain "*" \
+          --rpcvhosts=* \
+          --miner.gastarget 7500000 \
+          --ws --wsaddr 0.0.0.0 --wsorigins='*'
+    expose:
+      - 8545
+      - 8546
+      - 30303
+    ports:
+      - 9000:8545
+    volumes:
+      - ./data1:/data
+    healthcheck:
+      test: curl geth-1:8545
+      interval: 5s
+      timeout: 3s
+      retries: 5
+    networks:
+      chain:
+        ipv4_address: 172.25.0.102
+
+  geth-2:
+    image: ethereum/client-go:alltools-stable
+    depends_on:
+      - geth-1
+    hostname: geth-2
+    environment:
+      - account=0xc0f780dfc35075979b0def588d999225b7ecc56f
+      - bootnodes=enode://4574f825d67bf570b9216e704a5b761d05d5015c458e2c9dd4b30abb2fe8c881400c2074a126df94690c4c9fb72ee046e6e3ac2bb73dede42fce66cb0a963b36@172.25.0.102:30303
+    entrypoint:
+      - /bin/sh
+      - -c
+      - |
+        apk add --update curl
+        echo "" > /tmp/geth-password
+        echo $$account
+
+        geth --datadir data/ \
+          --bootnodes $$bootnodes \
+          --keystore=data/geth/keystore/ \
+          --networkid "1337" \
+          --gasprice "0" \
+          --unlock "0,1" \
+          --password /tmp/geth-password \
+          --mine --minerthreads=1 \
+          --etherbase $$account \
+          --nat extip:`hostname -i` \
+          --syncmode="full" \
+          --allow-insecure-unlock \
+          --rpc \
+          --rpcaddr "0.0.0.0" \
+          --rpcapi "personal,eth,web3,net,admin,debug,db" \
+          --rpccorsdomain "*" \
+          --rpcvhosts=* \
+          --miner.gastarget 7500000 \
+          --ws --wsaddr 0.0.0.0 --wsorigins='*'
+    expose:
+      - 8546
+      - 8545
+      - 30303
+    ports:
+      - 9001:8545
+    volumes:
+      - ./data2:/data
+    healthcheck:
+      test: curl geth-2:8545
+      interval: 5s
+      timeout: 3s
+      retries: 5
+    networks:
+      chain:
+        ipv4_address: 172.25.0.103
+
+  nginx:
+    image: nginx:latest
+    container_name: nginx
+    depends_on:
+      geth-1:
+        condition: service_healthy
+      geth-2:
+        condition: service_healthy
+    volumes:
+      - ./nginx.conf:/etc/nginx/nginx.conf
+    ports:
+      - 8545:80
+      - 8546:81
+      - 443:443
+    networks:
+      chain:
+        ipv4_address: 172.25.0.104
+    healthcheck:
+      test: curl nginx:80
+      interval: 5s
+      timeout: 3s
+      retries: 5
+
+  postgres:
+    image: postgres:9.6.13-alpine
+    restart: always
+    ports:
+      - "5432:5432"
+    environment:
+      POSTGRES_USER: omisego_dev
+      POSTGRES_PASSWORD: omisego_dev
+      POSTGRES_DB: omisego_dev
+    healthcheck:
+      test: pg_isready -U omisego_dev
+      interval: 5s
+      timeout: 3s
+      retries: 5
+    networks:
+      chain:
+        ipv4_address: 172.25.0.110
+
+  childchain:
```

Comment:
if you chain docker-compose files, you don't need to completely redefine them here. 
I think it's enough if you simply do:
```
childchain:
   environment:
       - ETHEREUM_RPC_URL=http://172.25.0.104:80
```
everything else is then merged with the default docker-compose.yml file.

Reply:
Will it redefine all properties including network?

Reply:
yeah, I think it merges and overwrites!

Reply:
@InoMurko I just tried overriding docker-compose file. It seems `depends_on` can be overridden.
https://github.com/docker/compose/issues/6426 It just merges values.

Reply:
what if we put nginx in docker-compose.yml?

Reply:
but it still will depend on geth. but in this docker compose geth-1 and geth-2 are used

Reply:
yeah, so we put nginx in docker-compose.yml in front of the single -non-reorg- geth.
and nginx here points to two geths, and in docker-compose.yml to single geth

Reply:
I added nginx to docker-compose.yml